.functions.php

```diff
@@ -77,8 +77,8 @@ function updateSession($sessionID, $connection){
 
 function openDatabaseConnection(){
   try{
-    require_once '.db_config.php';
-    $connection = new PDO("mysql:host=$dhost;dbname=$dname", $duser, $dpassword);
+    include '.db_config.php';
+    $connection = new PDO($dsn, $duser, $dpassword);
```

Comment:
I'm assuming the file was changed to match this...or maybe already had `$dsn` in it?

Reply:
I'm not sure what you're asking here

Reply:
Does `.db_config.php` contain the variable `$dsn`?

Reply:
Yes